### ISSUE TITLE
Jobboard

### ISSUE THREAD

**Issue by pmuens** [GravityProject/gravity#3]

**Comment by mattvella07** [comment]
@pmuens What are you thinking on this one, a new page where people can post/view open jobs?

**Comment by pmuens** [comment]
Yes. A new page would be great. I think at first a pretty easy CRUD job board is sufficient.

**Comment by mattvella07** [comment]
@pmuens I'll start working on this one, can you assign it to me?

**Comment by pmuens** [comment]
Great!
Hmm unfortunately you're not in the lest of people I can assign. But I think that someone else will read through this and see that you've started your work on it!

**Comment by zaverichintan** [comment]
Hey,  it's a nice feature to be implemented. 
I Would request you to use more user friendly UI so that with change in UI of other pages, it will not affect this job portal.

**Comment by mattvella07** [comment]
Done ✅

**Comment by pmuens** [comment]
@mattvella07 fantastic job! Thank you very much!
